src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -105,6 +116,11 @@
 
     private VariantContextWriter vcfWriter;
 
+    //todo: remove this when the reducible annotation framework is in place
+    //this is ok because RMS_MAPPING is stateless, but it needs to be instantiated in order to call its
+    //overridden instance methods
+    private static final RMSMappingQuality RMS_MAPPING_QUALITY = new RMSMappingQuality();
```

Comment:
can't override static methods :(  We could probably fix that problem if we move to scala...